mailviews/previews.py

```diff
@@ -13,8 +13,19 @@
 from django.core.urlresolvers import reverse
 from django.http import Http404
 from django.shortcuts import render
-from django.utils.datastructures import SortedDict
-from django.utils.importlib import import_module
+
+try:
+    # Django <= 1.8
+    from django.utils.datastructures import SortedDict as OrganizedDict
+except ImportError:
+    from collections import OrderedDict as OrganizedDict
```

Comment:
Any particular reason why you are preferring the deprecated Django versions here (initial import) instead of preferring the system versions (currently the fallback)?